syntax = "proto3";
package sv_rcon;
option optimize_for = LITE_RUNTIME;

enum response_t
{
	SERVERDATA_RESPONSE_AUTH        = 0;
	SERVERDATA_RESPONSE_CONSOLE_LOG = 1;
}

message response
{
	optional int32      messageID    = 1;
	optional int32      messageType  = 2;
    optional response_t responseType = 3;
    optional string     responseMsg  = 4;
    optional string     responseVal  = 5;
}
